test/integration/full-stack/serverspec/nginx_spec.rb

```diff
@@ -7,3 +7,11 @@
   end
 
 end
+
+describe file("/etc/nginx/sites-available/intercity_sample_app.conf") do
+  it { should be_file }
+
+  its(:content) do
```

Comment:
Can you but a `context` block around this? so it tells a bit more about what you're testing. So you get something like:
```
describe file("/etc/nginx/sites-available/intercity_sample_app.conf") do
  it { should be_file }

  context "GZip support enabled" do
    its(:content) do
      should match /location ~ \^\/\(assets\)\/.*gzip_static on;/m
    end
  end
end

```